lib/active_merchant/billing/gateways/paymentez.rb

```diff
@@ -262,7 +277,7 @@ def headers
       end
 
       def success_from(response)
-        !response.include?('error') && (response['status'] || response['transaction']['status']) == 'success'
+        !response.include?('error') && (SUCCESS_STATUS.include?(response['status'] || response['transaction']['status']) || response['status'] == 1)
```

Comment:
I like the way you've used a constant to define the statuses we are looking for 👍 

It seems like we also want to include `response['status'] == 0` just to be safe. I know the test transaction gives us `response['status']: pending` but the API documentation suggests that the value could also be `0`.

You could use the constant you have created to do this, and simplify the `success_from` method at the same time:

```
SUCCESS_STATUSES = ['success', 'pending', 1, 0]

def success_from(response)
  return false if response.include?('error')
 
  SUCCESS_STATUSES.include?(response['status'] || response['transaction']['status'])
end
```

Reply:
thanks, updated 👍